docs/pages/preview/moderated-sessions.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Moderated Sessions
+description: Session moderation with RBAC and humans
+h1: Moderated Sessions
+---
+
+This is an upcoming feature expected to land in Teleport 8.2.
+
+## Introduction
+
+Moderated sessions allows Teleport administrators to
+define requirements for other users to be present in a Server or Kubernetes Access session.
+
+Moderated Sessions is useful in the following scenarios:
```

Comment:
Nit: I'd say Moderated Sessions Feature is useful or Moderated sessions are useful